armada-ths/ais
# Exhibitor comment

Added "comment" field to Exhibitor.

Anyone who can edit individual exhibitors can edit the comment, it's possible to also add the comment to exhibitors view.

To test this:
* make sure you have at least one exhibitor
* Comment something on that exhibitor
* optinally, add "comment" to your exhibitors view and make sure you can see your comment.
